### PR TITLE
Geo blocklist - reimplement with backend support

### PR DESCRIPTION
**Pull request by infinite-persistence**
Closes [#1079 Support geoblocking channels/videos](https://github.com/OdyseeTeam/odysee-frontend/issues/1079)

## Changes
- Replaced the .env version with iapi version.
- Includes 'videos' blocking and custom messages.

### CODE REVIEW

File: ui/redux/selectors/claims.js [hunk 1]
```diff
@@ -829,3 +831,39 @@ export const selectOdyseeMembershipForChannelId = function (state: State, channe
...
+      // --- livestream
+      if (isStreamPlaceholderClaim(claim)) {
+        geoConfig = geoBlockLists.livestreams && geoBlockLists.livestreams[channelId];
+      }
+      // --- videos
+      else if (claim?.value?.video) {
+        geoConfig = geoBlockLists.videos && geoBlockLists.videos[channelId];
+      }
+
+      if (geoConfig) {
```
Comment (tzarebczan) [inline ui/redux/selectors/claims.js:854]: Ah, this came out so much cleaner :)

**Comment by infinite-persistence** [comment]
[810b9e1 ](https://github.com/OdyseeTeam/odysee-frontend/commit/810b9e1a609f7f4cd779c953e3a221f87ab0f1b8)to [976f6fc ](https://github.com/OdyseeTeam/odysee-frontend/commit/976f6fc44089d1a4de81f6367a89c16159212567)
- Updated logic to treat `videos` as "everything else"

**Comment by tzarebczan** [comment]
This looks good, thank you! I did notice a small issue where there's a large cursor in the blocked message. 
image

**Comment by infinite-persistence** [comment]
Any special sequence of file to try?
So far not seeing it on Brave, Firefox and Chrome Linux

image

**Comment by tzarebczan** [comment]
I think I clicked into the page on the left hand side. This is not a big deal at all...go to bed!

**Pull request merged**